A/B test 30 days limit release note (#867)
vtexdocs/dev-portal-content: docs/vtex-io/App-Guides/ab-tests/vtex-io-documentation-running-native-ab-testing.md

A/B testing involves comparing traffic between two store workspaces and helps you determine which one performs better in terms of user engagement and conversions.
To execute an A/B test, you can use the VTEX IO CLI or the [A/B Tester Admin app](https://developers.vtex.com/docs/guides/vtexarg-abtester).

> > ⚠️ A/B tests have a maximum duration of 30 days from its start. After this period, the test may end automatically without notice.

Reviewer: from their start*

Reviewer: trocar "the test" por "tests"

![ab-testing-finish-step4](https://vtexhelp.vtexassets.com/assets/docs/src/abtest-end___2940fdcba7933cab4d828ba9e7a34d72.png)

> > ❗ It is not possible to retrieve A/B test data after it is finished. Remember to save any relevant data before using the `vtex workspace abtest finish` command or it ends automatically in 30 days. You can access data from ongoing tests with the `vtex workspace abtest status` command.

Reviewer: ```suggestion
> ❗ Remember to save any relevant data using the `vtex workspace abtest finish` command before it ends automatically in 30 days – A/B test data cannot be retrieved otherwise. You can access data from ongoing tests with the `vtex workspace abtest status` command.
```
